src/app/shell/personal-cabinet/provider/provider-applications/provider-applications.component.ts

```diff
@@ -169,6 +170,8 @@ export class ProviderApplicationsComponent extends CabinetDataComponent implemen
   }
 
   onSendMessage(application: Application): void {
+    this.store.dispatch(new SetParent(application.parent));
```

Comment:
Chat is working now, but have a doubt about such solution.
It is needed to have parent object (id, name) to start chat. So here it is dispatching to store using parent **in application**.

Maybe there is another approach or it is better to fetch parent from backend?